Documentation/Configuration/TypoScript/Syntax/FileImports/Index.rst

```diff
@@ -150,6 +150,10 @@ More details:
 
   Files are included in alphabetical. Also files are included first, then directories.
 
+  :typoscript:`<INCLUDE_TYPOSCRIPT:` with :typoscript:`DIR:` and relative
```

Comment:
As this can be quire breaking I would put an admonition like warning here?

Reply:
I thought about it, yes, maybe...

Reply:
Done